catalyst/metrics/region_base_metrics.py

```diff
@@ -0,0 +1,204 @@
+from typing import Callable, List, Optional, Union
+from functools import partial
+
+import torch
+
+from catalyst.utils.torch import get_activation_fn
+
+
+def _get_segmentation_stats(
+    outputs: torch.Tensor,
+    targets: torch.Tensor,
+    class_dim: Optional[int] = None,
+    threshold: float = None,
+    activation: str = "Sigmoid",
```

Comment:
let's remove `activation` from all metrics
it's better to use extra wrappers for activations and work with correct `scores` and `targets` for metrics computation